runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -45,8 +45,8 @@ processResources {
   filter org.apache.tools.ant.filters.ReplaceTokens, tokens: [
     'dataflow.legacy_environment_major_version' : '8',
     'dataflow.fnapi_environment_major_version' : '8',
-    'dataflow.legacy_container_version' : 'beam-master-20210913',
-    'dataflow.fnapi_container_version' : 'beam-master-20210913',
+    'dataflow.legacy_container_version' : 'beam-master-20211213',
+    'dataflow.fnapi_container_version' : 'beam-master-20211213',
```

Comment:
I think java fnapi beam-master image  is a clone of an image released externally as snapshot of Beam master, so  we need to clone it into internal GCR registry first.
It would make sense to do after you upgrade it to bullseye (externally) first.

cc: @emilymye

Reply:
yes - in this case I think it makes sense to update the legacy and FnAPI containers tags seperately, as they correspond to two different releases internally.